Updated Prometheus Metrics Documentation (#4410)
opensearch-project/documentation-website: _observing-your-data/prometheusmetrics.md

You can view metrics collected from Prometheus in OpenSearch Dashboards by first creating a connection from [Prometheus](https://prometheus.io/) to OpenSearch using the SQL plugin. 

> To configure a connection to Prometheus, one can use datasource configuration APIs. 

Reviewer: ```suggestion
To configure a connection to Prometheus, you can use the `_datasources` configuration API endpoint. 
```

Example Request for Prometheus datasource with no authentication.

```json
POST _plugins/_query/_datasources 
{
    "name" : "my_prometheus",
    "connector": "prometheus",
    "properties" : {
        "prometheus.uri" : "http://localhost:9090"
    }
}
```

> Example Request for Prometheus datasource with AWS SigV4 authentication:

Reviewer: ```suggestion
The following example request configures a Prometheus data source with AWS SigV4 authentication:
```

        "prometheus.auth.access_key" : "{{accessKey}}"
        "prometheus.auth.secret_key" : "{{secretKey}}"
    }
}
```

After configuring the connection from Prometheus to OpenSearch, Prometheus metrics are displayed in Dashboards in the **Observability** > **Metrics analytics** window, as shown in the following image.

![Metrics UI example 1]({{site.url}}{{site.baseurl}}/images/metrics/metrics1.png)

> * For more information related to authn/authz of datasource APIs, see the [Datasource](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/admin/datasources.rst) GitHub Page.

Reviewer: ```suggestion
* For more information about authentication and authorization of data source APIs, see [data source documentation on GitHub](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/admin/datasources.rst).
```
